website/addons/owncloud/static/owncloudUserConfig.js

```diff
@@ -0,0 +1,226 @@
+
+
+var ko = require('knockout');
+var $ = require('jquery');
+var Raven = require('raven-js');
+var bootbox = require('bootbox');
+require('js/osfToggleHeight');
+
+var language = require('js/osfLanguage').Addons.owncloud;
+var osfHelpers = require('js/osfHelpers');
+var addonSettings = require('js/addonSettings');
+
+var ExternalAccount = addonSettings.ExternalAccount;
+
+var $modal = $('#ownCloudCredentialsModal');
+
+
+function ViewModel(url) {
```

Comment:
Hinted at [elsewhere](https://github.com/CenterForOpenScience/osf.io/pull/6075/files/b9fcb57ebb58841535a051209878dbcb51edbcac#r73215120), but with a small model change this could probably be simplified by `oop.extend`ing `OAuthAddonSettingsViewModel` in `website/static/js/addonSettings.js`